### PR TITLE
feat(@vtmn/svelte): add `VtmnChip` component

### PR DESCRIPTION
**Pull request by Tlahey**
Create a svelte `VtmnChip` component.

Features : 
- Add README.md on the root component `VtmnChip`
- Add component test thanks to jest :) 
- Add new libraries for testing on `packages/sources/svelte`
- Add task test on svelte build and project package.json
- Add more than 44 units tests on the `VtmnChip` component
- `VtmnChip` import component `VtmnBadge` for the filter badge and `VtmnButton` for the action variant

### CODE REVIEW

File: packages/sources/svelte/src/components/selection-controls/VtmnChip/VtmnChip.svelte [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      {disabled}
+    />
+  {/if}
+  {#if displayFilterBadge}
+    <VtmnBadge value={badgeValue} />
+  {/if}
+</div>
+
+<style lang="css">
+  @import '@vtmn/css-chips';
```
Comment (GaspardMathon) [inline packages/sources/svelte/src/components/selection-controls/VtmnChip/VtmnChip.svelte:98]: I think it is `@vtmn/css-chip`

**Review by GaspardMathon — CHANGES_REQUESTED** [review]

File: packages/sources/svelte/src/components/selection-controls/VtmnChip/VtmnChip.svelte (hunk 1 above)
Comment (Tlahey) [reply packages/sources/svelte/src/components/selection-controls/VtmnChip/VtmnChip.svelte:98]: Ho yes ! Sorry xD

**Comment by Tlahey** [comment]
I have a question though.
Wouldn't it be better to make X components Chip for each variant. So, each one has the necessary props and would not share props not useful for the variant 🤔 

For example, only chip filter can have a badge, so if we create a `VtmnChipFilter` it's more clear to see props used by the component

In counterpart, we have more component to maintains, but it's more clear ... To discuss.

File: packages/sources/svelte/src/components/selection-controls/VtmnChip/enums.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export const VARIANTS = {
```
Comment (lauthieb) [inline packages/sources/svelte/src/components/selection-controls/VtmnChip/enums.js:1]: => `VTMN_CHIP_VARIANT`

Same for size :)

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks a lot @Tlahey, small changes :)

**Comment by lauthieb** [comment]
+ please delete `package-lock.json` as we only use yarn.

**Comment by lauthieb** [comment]
@Tlahey I understand your proposition but we need to be as near as possible from the documentation and Figma, that's why I prefer only one component, even if some variants cannot be present together... Thanks for your understanding

**Review by lauthieb — APPROVED** [review]
Thanks a lot @Tlahey :)

**Pull request merged**